src/components/organisms/cards/cards.scss

```
@@ -0,0 +1,17 @@
+@use "../../variables" as *;
+.cards {
+    display: flex;
+    justify-content: space-between;
+    padding-bottom: 100px;
+    @media (max-width: $breakpoint-md) {
+        display: block;
+        padding-bottom: 0;
+    }
+    .card {
+        width: 31%;
+        @media (max-width: $breakpoint-md) {
+            width: 100%;
+            margin-bottom: 40px;
+        }
+    }
```

```suggestion
    padding-bottom: 100px;
    gap: 24px;
    @media (max-width: $breakpoint-md) {
        display: block;
        padding-bottom: 0;
        gap: 40px
    }
    .card {
        flex: 1;
    }
```
gapとflexを使ったほうがスッキリ書けるのでおすすめです。
gap などについては新課題の方で軽く触れているので参考になるかもです。
https://shinonome-inc.notion.site/Flexbox-1-e28e57a82d004507bfef1179b8fd6845